helper/keysutil/policy.go

```diff
@@ -205,6 +205,60 @@ func (kem deprecatedKeyEntryMap) UnmarshalJSON(data []byte) error {
 // keyEntryMap is used to allow JSON marshal/unmarshal
 type keyEntryMap map[string]KeyEntry
 
+// PolicyConfig is used to create a new policy
+type PolicyConfig struct {
+	// The name of the policy
+	Name string `json:"name"`
+
+	// The type of key
+	Type KeyType
+
+	// Derived keys MUST provide a context and the master underlying key is
+	// never used. If convergent encryption is true, the context will be used
+	// as the nonce as well.
+	Derived              bool
+	KDF                  int
+	ConvergentEncryption bool
```

Comment:
If possible we shouldn't allow this to be set; it should always be 2. However if you plan to use this after loading from disk maybe that's not possible to take it out?

Reply:
Updated to make sure it's always 2 if convergent encryption is enabled

Reply:
This should only be used to create new policies, so we shouldn't need to set it to any other value